py/server/deephaven/plot/figure.py

```diff
@@ -503,6 +506,9 @@ def chart(
             non_null_args = non_null_args.difference({"pie_label_format"})
             f_called = True
 
+        if multi_series_key_used:
+            non_null_args = non_null_args.difference({"multi_series_key"})
```

Comment:
Adding comments explaining why this is necessary would be nice as it is not obvious from the code itself.

Reply:
Will add comment after Chip reviews

Reply:
I think the comment in the generator explains things: `// prevent removal of multi_series_key until after it's been fully used`.  @jmao-denver do we need more?

Reply:
No. That should be enough.